src/chrome/content/rules/Steamrep.com.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="Steamrep.com">
+  <target host="steamrep.com" />
+  <target host="www.steamrep.com" />
+
+  <rule from="^http://(www\.)?steamrep\.com/" to="https://www.steamrep.com/" />
```

Comment:
https://www.steamrep.com works fine (i.e. no SSL error). [2d1](https://github.com/EFForg/https-everywhere/pull/2083#discussion_r34428201) prefer not to preempting server's redirect.